packages/cli/src/controllers/mfa.controller.ts

```diff
@@ -52,6 +62,8 @@ export class MFAController {
 		const { token = null } = req.body;
 		const { id, mfaEnabled } = req.user;
 
+		await this.externalHooks.run('mfa.beforeSetup', [req.user]);
```

Comment:
Adding the check here to avoid users bypassing the check by calling the API directly.